src/js/weather.js

```diff
@@ -0,0 +1,94 @@
+import { renderSearch } from './render';
+
+const weatherApp = (params) => {
+  const { city, country, units } = params;
+  const url = 'https://api.openweathermap.org/data/2.5/forecast?q=';
+  const id = '&APPID=092878e572bdc25b4b5bdea6cbd439db';
+  const deg = units === 'farenheit' ? '&units=imperial' : '&units=metric';
+  const forecastPromise = fetch(`${url}${city},${country}${deg}${id}`, { mode: 'cors' });
```

Comment:
`cors` is going to be the default in this case, so no need to explicitly set the `mode`.

Reply:
got it